index.md

```diff
@@ -64,6 +64,12 @@ Some of them, like `scala-js-dom` and `scala-js-jquery`, are specific to Javascr
 Some others, such as `scala-async`, are pure-macro projects, and thus work with Scala.js out of the box.
 Most of them, though, are cross-compiling over JVM and JS, and thus have separate artifacts to run on each platform.
 
+If you have a library that would be useful to the Scala.js community, please add it to this list! 
+[This link](https://github.com/scala-js/scala-js-website/edit/master/index.md) takes you directly to the 
```

Comment:
Links saying "click here" are in general considered bad practice. Just make "add it to this list" a link and omit the next sentence entirely.